SPHINXsys/src/shared/materials/elastic_solid.h

```diff
@@ -172,6 +172,49 @@ namespace SPH {
 		virtual Real VolumetricKirchhoff(Real J) override;
 	};
 
+	//preparation for the implementation of the aorta material model
+	/**
+	* @class OrthotropicSolid
+	* @brief Ortothropic solid
+	*/
+	class OrthotropicSolid : public LinearElasticSolid
+	{
+	public:
+		OrthotropicSolid() : LinearElasticSolid() 
+		{
+			material_name_ = "OrthotropicSolid";
+		};
+		OrthotropicSolid(Real rho_0, std::array<Vecd, 3> a, std::array<Real, 3> E, std::array<Real, 3> G,std::array<Real, 3> poisson)
```

Comment:
we have to call all these functions in the constructor here:
CalculateAllMu();
CalculateAllLambda();
CalculateA0();

Reply:
and we should also move the constructor to the .cpp and only declare it here